lib/server/loop.js

```diff
@@ -81,6 +81,9 @@ function init (env, ctx) {
          if (data.otp !== undefined && data.otp.length > 0) {
             payload["otp"] = ""+data.otp
          }
+        if (data.created_at !== undefined) {
+          payload['created-at'] = data.created_at;
```

Comment:
I see that NS is referring this to as "created-at", but a better name for the Loop side of things would be "start-time", since Loop uses "created at" as the time of entry, which can be different from the time of eating, which is start time.

Reply:
This has been updated to "start-time". Thanks.

Reply:
Just a note - There is an problem on dev that is causing dev to be not deployable on Heroku: https://github.com/nightscout/cgm-remote-monitor/issues/7475

I've worked around that by testing with the branch that Ben has which is linked in that case in another PR.

I'm mentioning as whomever wants to try this change, that uses Heroku, will need to cherry-pick it to a NS branch that is working like master or Ben's branch.

Reply:
Thanks for contributing!  Cherry-pick shouldn't be needed. We'll merge this in.